docs/console-virtual-terminal-sequences.md

```diff
@@ -107,6 +107,25 @@ The following commands control the visibility of the cursor and its blinking sta
 > [!TIP]
 > The enable sequences end in a lowercase H character (`h`) and the disable sequences end in a lowercase L character (`l`).
 
+## <span id="Cursor_Shape"></span><span id="cursor_shape"></span><span id="CURSOR_SHAPE"></span>Cursor Shape
+
+
+The following commands control and allow for customization of the cursor shape.
+
+
+| Sequence | Code | Description | Behavior |
+|---------------|---------|------------------------------|---------------------------|
+| ESC \[ 0 SP q | DECSCUSR | User Shape | Default cursor shape configured by the user |
+| ESC \[ 1 SP q | DECSCUSR | Blinking Block | Blinking block cursor shape |
+| ESC \[ 2 SP q | DECSCUSR | Steady Block | Steady block cursor shape |
+| ESC \[ 3 SP q | DECSCUSR | Blinking Underline | Blinking underline cursor shape |
+| ESC \[ 4 SP q | DECSCUSR | Steady Underline | Steady underline cursor shape |
+| ESC \[ 5 SP q | DECSCUSR | Blinking Bar | Blinking bar cursor shape |
+| ESC \[ 6 SP q | DECSCUSR | Steady Bar | Steady bar cursor shape |
+
+> [!NOTE]
+> `SP` is a literal space character of 0x20 as an Intermediate Byte, followed by `q` which is 0x71 as a Final Byte. 
```

Comment:
```suggestion
> `SP` is a literal space character (0x20) in the intermediate position, and it is followed by `q` (0x71) in the final position. 
```

I slightly reworded what @KalleOlaviNiemitalo said in the linked issue; feel free to accept or reject this change. :)